examples/multisource_adapt/config.py

```diff
@@ -0,0 +1,72 @@
+"""
+Default configurations for multi-source domain adapation
+"""
+import os
+
+from yacs.config import CfgNode as CN
+
+# import os
+
+# -----------------------------------------------------------------------------
+# Config definition
+# -----------------------------------------------------------------------------
+
+_C = CN()
+
+# -----------------------------------------------------------------------------
+# Dataset
+# -----------------------------------------------------------------------------
+_C.DATASET = CN()
+_C.DATASET.ROOT = "../data"
+_C.DATASET.NAME = "digits"  # dset choices=['office', 'image-clef', 'office-home']
+# _C.DATASET.SOURCE = ["cartoon", "art_painting", "photo"]
```

Comment:
no default source? It seems the default configs won't run (w/o specifying a default source)

Reply:
do you imply the sources?

Reply:
what if I just want to use two sources (faster, e.g. for notebooks) rather than all three?